plugin/nvimgdb.vim

```diff
@@ -3,6 +3,10 @@ if exists("g:loaded_nvimgdb") || !has("nvim")
 endif
 let g:loaded_nvimgdb = 1
 
+if !&scrolloff
```

Comment:
I believe this isn't the best place for the option. We'd like to have it local to the window where the code is going to be displayed. The window is created in the function `nvimgdb#Spawn()`.